package.json

```diff
@@ -65,9 +70,7 @@
     "karma": "^6.3.7",
     "karma-chrome-launcher": "^3.1.0",
     "karma-jasmine": "^4.0.1",
-    "karma-requirejs": "^1.1.0",
```

Comment:
Tests pass without this, so I think we can remove requirejs from the dev dependencies for good. No more need for almond either.